Makefile

```diff
@@ -95,6 +95,11 @@ tests:
 
 vet:
 	go vet ./...
+	go tool vet --shadow ./test
```

Comment:
I am sorry about all the bickering, but I feel that when tools are in an experimental phase they cause pain when they are built into a process.
The pain is only worth it, if this is contributing back to the experiment, which I don't think it is.
The tool is valuable and finds bugs, as you have shown, but it requires human intervention, at this point, that is why it will be great to run it manually.
Please take your time and reconsider.